uwblueprint/ccbc
# Delete review from admin dashboard

## Notion ticket link

[Delete Review from Admin Dashboard](https://www.notion.so/uwblueprintexecs/Delete-Review-from-Admin-Dashboard-3ff511af5a7246d09b0b38cc48861ea0)



## Implementation description
* When delete icon in a row is clicked, open confirmation dialog
* Clicking delete review removes the review from data and removes the row
* Clicking go back does not delete the review



## Steps to test
1. Delete review, and verify that the row disappears
2. Refresh the page to verify that the removed review is deleted
3. Go back on confirmation dialog and verify that the row does not disappear
4. Refresh the page to verify that the review was not removed



## What should reviewers focus on?
* code & correctness


## Checklist
